app/views/pages/home.html.haml

```diff
@@ -139,7 +139,7 @@
     .small-11.medium-6.small-centered.columns
       / MailChimp Signup Form
       #mc_embed_signup
-      %form#mc-embedded-subscribe-form.validate{action: "//hackedu.us10.list-manage.com/subscribe/post?u=738aabcd5284a4d3df761a981&id=4c8d1b3833",
+      %form#mc-embedded-subscribe-form.validate{action: "//hackclub.io10.list-manage.com/subscribe/post?u=738aabcd5284a4d3df761a981&id=4c8d1b3833",
```

Comment:
Classes should be listed before IDs (.validate should precede #mc-embedded-subscribe-form)<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line is too long. [147/80]